.github/workflows/deploy_to_aws.yml

```
@@ -47,507 +47,125 @@
+          terraform init -backend-config="environments/backend-${{ vars.ENV_NAME }}.hcl"
+          terraform plan  -out="${{ vars.ENV_NAME }}.tfplan"
         env:
-          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-          AWS_DEFAULT_REGION: "eu-west-1"
-          SETTINGS_BUCKET: oshub-settings-${{ steps.get_env_name.outputs.lowercase }}
-
+          AWS_ACCESS_KEY_ID: ${{ secrets.TF_AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.TF_AWS_SECRET_ACCESS_KEY }}
+          AWS_DEFAULT_REGION: "us-east-1"
+   
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)